test_lib/compaction.py

```diff
@@ -23,6 +42,36 @@ def from_str(cls, output_str):
             raise ValueError(err_msg) from attr_err
 
 
+def get_gc_mode(node: BaseNode, keyspace: str, table: str) -> str | GcMode:
+    """Get a given table GC mode
+
+    :Arguments:
+        node
+        keyspace
+        table
+    """
+    table_gc_mode_result = node.run_cqlsh(
+        f'SELECT extensions FROM system_schema.tables where keyspace_name = {keyspace} and table_name = {table}',
+        split=True)
+    LOGGER.debug("Query result for %s.%s GC mode is: %s", keyspace, table, table_gc_mode_result)
+    gc_mode = 'N/A'
+    if table_gc_mode_result and len(table_gc_mode_result) >= 4:
+        extensions_value = table_gc_mode_result[3]
+        # TODO: A temporary workaround until 5.0 query-table-extensions issue is fixed:
+        # https://github.com/scylladb/scylla/issues/10309
+        if '6d6f646506000000' in extensions_value:
```

Comment:
In fact, setting `6d6f646506000000` as value for `REPAIR` etc. would do the job without `if` statements.

Reply:
Correct, but better keep the Enum class in the right form with correct variables. So eventually, only this section will require a rewrite.

Reply:
without this section, only Enum class would require a rewrite...